### PR TITLE
Improve query options document

### PR DESCRIPTION
**Pull request by jpg-130**
Added documentation to specify multiple query options in REST API

### CODE REVIEW

**Review by Jackie-Jiang — COMMENTED** [review]
Thanks for help improve the documentation. Do you use query parameter to set query options? We are considering whether we should remove it and only allow using `SET` in the future

**Comment by jpg-130** [comment]
I used it just to keep the code clean. 
However, it is also fine if that option is removed altogether as we have two different ways of doing the same thing.

**Comment by Jackie-Jiang** [comment]
Since we support this right now, we can still add it, but maybe mark it deprecated? Actually this was used before `OPTIONS` was introduced.
We also support this format within the broker API (`PinotClientRequest`). Could you please also add that?

File: users/user-guide-query/query-options.md [hunk 1]
```diff
@@ -168,3 +168,25 @@ SELECT * FROM myTable OPTION(key1=value1, key2=123)
...
+```
+* [Using Broker Query API](../../users/api/querying-pinot-using-standard-sql/README.md)
+```bash
+curl -X POST 'http://localhost:8099/query/sql' \
+-d '{
+  "sql": "SELECT * FROM myTable;",
+  "trace": false,
+  "queryOptions":"key1=value1;key2=123"
+}'
+```
```
Comment (Jackie-Jiang) [inline users/user-guide-query/query-options.md:192]:
```suggestion

### REST API

Query options can be specified in API using queryOptions as key and **';'** separated key-value pairs.  
Alternatively, we can also use the SET keyword in the sql query.

* [Using Controller Admin API](../../users/api/pinot-rest-admin-interface.md)

```bash
curl -X POST 'http://localhost:9000/sql' \
-d '{
  "sql": "SELECT * FROM myTable",
  "trace": false,
  "queryOptions":"key1=value1;key2=123"
}'
```

* [Using Broker Query API](../../users/api/querying-pinot-using-standard-sql/README.md)

```bash
curl -X POST 'http://localhost:8099/query/sql' \
-d '{
  "sql": "SELECT * FROM myTable;",
  "trace": false,
  "queryOptions":"key1=value1;key2=123"
}'
```
```

**Pull request merged**